controls/V-219180.rb

```diff
@@ -1,4 +1,6 @@
 # encoding: UTF-8
+min_reuse_generations = input('min_reuse_generations', value: 5,
```

Comment:
same as above, use the `inspec.yml inputs:` section for this, and then call the input directly in the control.